libs/@local/hash-backend-utils/src/wal2json.ts

```diff
@@ -5,15 +5,15 @@
  * Note: We ignore transaction begin / end messages (i.e. those with action "B" / "C"),
  * and TRUNCATE messages (action "T").
  */
-export type Wal2JsonMsg = {
+export type Wal2JsonMsg<T = string> = {
```

Comment:
Should this be `T extends string`?

Reply:
Yes, thanks! 631acd495b532c5a0de962e09db9974d0fd1ca0e